package.json

```diff
@@ -39,7 +39,7 @@
     "git-revision-webpack-plugin": "^3.0.3",
     "html-webpack-plugin": "^3.2.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.6.0",
+    "jest": "^24.8.0",
     "jest-cli": "^24.6.0",
```

Comment:
Was this intentional? Don't think it really matters but not sure what the  protocol is on updating major packages in our code base is... (cc: @ebdabbs) If we are updating this, I would recommend keeping the `jest-cli` package on the same version!

Reply:
Prolly left over from getting the tests to work